proto/ignition/msgs/wrench_visual.proto

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+package ignition.msgs;
+option java_package = "com.ignition.msgs";
+option java_outer_classname = "WrenchVisualProtos";
+
+/// \ingroup ignition.msgs
+/// \interface Wrench Stamped
+/// \brief Message used to visualize a wrench applied to an entity.
+
+import "ignition/msgs/entity.proto";
+import "ignition/msgs/header.proto";
+import "ignition/msgs/wrench.proto";
+import "ignition/msgs/vector3d.proto";
+
+message WrenchVisual
+{
+  /// \brief Optional header data
+  Header header                     = 1;
+
+  /// \brief The entity transmitting the data
+  Entity entity                     = 2;
+
+  /// \brief Label that will be visible int he GUI panel
+  string label                      = 4;
+
+  /// \brief The actual wrench
+  ignition.msgs.Wrench wrench       = 5;
+
+  /// \brief Location where the wrench is applied.
```

Comment:
We should document the reference frame and units for the wrench and the position